zipline/protocol.py

```diff
@@ -242,6 +247,7 @@ def __repr__(self):
     __getitem__ = _deprecated_getitem_method(
         'position', {
             'sid',
+            'asset',
```

Comment:
now the user-facing `Position` object will have both `sid` and `asset` properties.  is that OK?